app/src/main/res/values/strings.xml

```diff
@@ -66,6 +66,53 @@
         https://www.reddit.com/r/SimpleMobileTools
     </string>
 
+    <!-- GUI -->
+    <string name="settings">Settings</string>
```

Comment:
why are you readding these strings if they are in Simple Commons already?

Reply:
Sorry, I'm added those for the language support. Should I add them in the specific language file?